src/vimish/commands/connect.js

```diff
@@ -1,13 +1,8 @@
 const Command = require("../command")
-const Bus = require("../../bus")
 const Autodetect = require("../../autodetect")
+const redraw = require("../../util/redraw")
```

Comment:
this needs to be a destructed argument: `{redraw}` since you are namespacing it in the redraw util

Reply:
Is that the correct term for this syntax? It looks like this is declaring a const set, but that's drawing from other languages, so I don't think this is the right read. What term should I look up to better understand how to pull things from a namespaced module?

Reply:
Yeah, here is a good over-view for you [Destructuring Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)